src/components/Footer/Footer.css

```diff
@@ -0,0 +1,76 @@
+.footer {
+    position: relative; /* Ensure positioning for pseudo-element */
+    display: flex;
+    justify-content: space-between;
+    padding: 20px;
+    background-color: #f8f8f8;

```

Comment:
Please use the colors used in `theme.tsx`.
For example, 
``` css
 background-color: theme.colors.brightBackground;
```

Reply:
Do u want me to create .scss file? .css file can't load .tsx file components.